Phonebook.TwilightSaw/Factory/HostFactory.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.EntityFrameworkCore;
+using TwilightSaw.Phonebook.Controller;
+
+namespace TwilightSaw.Phonebook.Factory
+{
+    internal class HostFactory
+    {
+        public static IHost CreateDbHost(string[] args)
+        {
+            return Host.CreateDefaultBuilder(args)
+                .ConfigureServices((context, services) =>
+                {
+                    var configuration = context.Configuration;
+
+                    services.AddDbContext<AppDbContext>(options =>
+                        options.UseSqlServer(configuration.GetConnectionString("DefaultConnection")));
```

Comment:
🟠 **Code Duplication**

💡 I am so confused, are you doing this here or are you doing it in `AppDbContext.cs`